pkg/server/endpoints/config.go

```diff
@@ -61,7 +62,12 @@ type Config struct {
 	// Makes policy decisions
 	AuthPolicyEngine *authpolicy.Engine
 
-	Log     logrus.FieldLogger
+	// The fielLogger interface
+	Log             logrus.FieldLogger
+
+	// The default (original config) log level
+	DefaultLogLevel logrus.Level
```

Comment:
This may not be needed (see my comment in the logger service implementation)

Reply:
It is needed, but mostly because it is needed to be passed through the mux to the logger service.

The actual Server config isn't passed through, so I can't pull the value from the Server config in the logger service when I need to handle the ResetLogLevel RPC call.   

The current pattern is to pass portions of the Server config to the various services that need them, and not to pass the entire Server config.  While it might be better to actually pass the entire server config, that's a much larger change, and should be in its own PR.

Reply:
This is fine.